lib/segment/src/segment_constructor/segment_constructor_base.rs

```diff
@@ -376,9 +376,11 @@ pub(crate) fn create_vector_index(
     stopped: &AtomicBool,
 ) -> OperationResult<VectorIndexEnum> {
     let vector_index = match &vector_config.index {
-        Indexes::Plain {} => {
-            VectorIndexEnum::Plain(PlainIndex::new(id_tracker, vector_storage, payload_index))
-        }
+        Indexes::Plain {} => VectorIndexEnum::Plain(PlainVectorIndex::new(
+            id_tracker,
+            vector_storage,
+            payload_index,
+        )),
```

Comment:
I don't get why this changed the formatting, am I missing something? 🦮

Reply:
the struct name went from `PlainIndex` to `PlainVectorIndex` which makes the line longer